hometask-1/src/counterType2/counterType2.js

```diff
@@ -0,0 +1,20 @@
+import React, {Component} from 'react'
+
+export default  class CounterType2 extends Component {
+  state = {
+    value: 0
+  }
+
+  handleDecrease = () => this.setState(state => ({value: state.value -= 1}))
```

Comment:
`this.setState(state => ({...state, value: state.value -= 1}))`
You are overriding a whole state each time
let's imagine that somewhere you add something to your locale state